xcm/pallet-xcm-benchmarks/src/fungible/benchmarking.rs

```diff
@@ -233,7 +233,7 @@ benchmarks_instance_pallet! {
 	}: {
 		executor.execute(xcm)?;
 	} verify {
-		if let Some(checked_account) = T::CheckedAccount::get() {
+		if T::TrackBalances::get() && let Some(checked_account) = T::CheckedAccount::get() {
 			// teleport checked account should have received some asset.
 			assert!(!T::TransactAsset::balance(&checked_account).is_zero());
```

Comment:
Can you explain the scenario where the balance is not tracked? Like why would the tracking not happen?

Reply:
Cumulus does not track balances teleports.

Reply:
But isn't it true that you can safely assume that the existence/non-existence of the checked account implies that tracking/no tracking is necessary? Trying to figure out why we need another Boolean configurable here.

Reply:
Turned out that currently we don't, because we aren't using the second benchmark in cumulus either.

Reply:
but more so, if we did care about the checked account being present or not, we should be able to pull it out of the config of xcm, rather than us re-declaring it again